Source/Server/Properties/launchSettings.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "profiles": {
+    "Server": {
+      "commandName": "Project",
+      "dotnetRunMessages": "true",
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:5000",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
Does this mean that we now have to explicitly set the launchSettings or the environmental variable when we deploy it in the platform?

Reply:
The default behavior of Asp.Net Core is to run with http://*:80 - so we don’t need this when running.